server/controllers/loginSignup.js

```diff
@@ -16,16 +15,16 @@ router.post('/login', (req, res) => {
     } else {
     //check if password matches the ones in database, consider HASH
       if (response[0].password === password) {
```

Comment:
Do we currently disallow many users to have the same username? If so we can use pg-promise one method to ensure we only get one user